eZ/Publish/Core/Repository/Helper/FieldTypeRegistry.php

```diff
@@ -44,7 +44,7 @@ public function getFieldTypes()
         {
             if ( !$value instanceof SPIFieldType )
             {
-                return $this->getFieldType( $identifier );
+                $this->getFieldType( $identifier );
```

Comment:
maybe offtopic, but a bit weird having a ```get``` function acting as if it doesn't return anything, don't you think?